Assignments/Week1/WaveArray.py

```diff
@@ -0,0 +1,12 @@
+class Solution:
+    def wave(self, A):
+        A.sort()
+        i=0
+        while(i<len(A)):
+            if i+1<len(A):
+                temp=A[i]
+                A[i]=A[i+1]
```

Comment:
In python, swapping can be done in this way: A[i], A[i+1] = A[i+1], A[i]. Much better code. Use this.